packages/starlight/__tests__/basics/config-errors.test.ts

```diff
@@ -122,8 +122,7 @@ test('errors with bad social icon config', () => {
 		"[AstroUserError]:
 			Invalid config passed to starlight integration
 		Hint:
-			**social.unknown**: Invalid enum value. Expected 'twitter' | 'mastodon' | 'github' | 'gitlab' | 'bitbucket' | 'discord' | 'gitter' | 'codeberg' | 'codePen' | 'youtube' | 'threads' | 'linkedin' | 'twitch' | 'azureDevOps' | 'microsoftTeams' | 'instagram' | 'stackOverflow' | 'x.com' | 'telegram' | 'rss' | 'facebook' | 'email' | 'reddit' | 'patreon' | 'signal' | 'slack' | 'matrix' | 'openCollective' | 'hackerOne' | 'blueSky' | 'discourse' | 'zulip' | 'pinterest' | 'tiktok' | 'nostr' | 'backstage' | 'farcaster' | 'confluence' | 'jira' | 'storybook' | 'npm' | 'sourcehut' | 'substack', received 'unknown'
-			**social.unknown**: Invalid url"
+			**social**: Expected type \`"array"\`, received \`"object"\`"
```

Comment:
I guess technically the error map could for the `social` error path detect if the received type is `object` and in this case, show a more helpful message like the changeset altho I think considering how small the syntax change is and well described in the changelog, it's fine to keep it simple?

Reply:
Very good idea. Added this in 69ed1a68 🙌